docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -548,9 +548,16 @@ The {ref}`sql-schema-table-management` functionality includes support for:
 #### Schema evolution
 
 Iceberg supports schema evolution, with safe column add, drop, reorder, and
-rename operations, including in nested structures. Table partitioning can also
-be changed and the connector can still query data created before the
-partitioning change.
+rename operations, including in nested structures. 
+
+Iceberg supports updating column types only for widening operations:
+ 
+- `INTEGER` to `BIGINT`
```

Comment:
are those Trino or Iceberg types?

Reply:
Yes, see the Iceberg spark docs for reference https://iceberg.apache.org/docs/latest/spark-ddl/#alter-table--alter-column and also the test code  https://github.com/trinodb/trino/pull/15651/files#diff-6be05909e810c0224c1951c4102cad6256e9e088feb91e4c260756bfde89b6d5R2254-R2260